zebra/zebra_l2.c

```diff
@@ -241,19 +243,22 @@ void zebra_l2if_update_bridge_slave(struct interface *ifp,
 {
 	struct zebra_if *zif;
 	ifindex_t old_bridge_ifindex;
+	struct zebra_vrf *zvrf;
 
 	zif = ifp->info;
 	assert(zif);
 
+	zvrf = zebra_vrf_lookup_by_id(ifp->vrf_id);
+	assert(zvrf);
```

Comment:
Should this be an assert or a simple return on zvrf being NULL ?

Reply:
this call comes from netlink, so I don't think we can have cases where zvrf is not found.
however, to be safe, I will change the check.